radixsort.py

```diff
@@ -0,0 +1,60 @@
+#! /usr/bin/env python
+import math
+import random
+import time
+
+
+def timed_func(func):
+    """Decorator for timing our insertion sort methods."""
+    def timed(*args, **kwargs):
+        start = time.time()
+        result = func(*args, **kwargs)
+        elapsed = time.time() - start
+        print "time expired: %s" % elapsed
+        return result
+    return timed
```

Comment:
Have you noticed how the `timed_func` decorator is reproduced in every sorting algorithm file you've created?  Can you think of a way to reduce that repetition?